docker-compose.yml

```diff
@@ -21,32 +21,52 @@ slave:
         - jenkins
     volumes:
         - ./slave:/home/slave
+        - ./nginx/conf.d:/home/slave/nginx
     volumes_from:
         - git
+        - nginx
     environment:
         - JAVA_HOME=/etc/alternatives/java_sdk
         - SERVICE_NAME=SPACENAME-omero
         - DOCKER_FIX='   '
-    command: -labels java -labels slave -name slave -executors 4
+    command: -labels java -labels slave -name slave -disableClientsUniqueId -executors 4
 
 web:
     build: web
-    environment:
-        - JENKINS_LABEL=/etc/alternatives/java_sdk
-        - JENKINS_MODE=exclusive
-        - SERVICE_NAME=SPACENAME-omero
+    ports:
+        - "4080:4080"
```

Comment:
This will need to be in the overwrite file (e.g. docker-compose.osx.yml) since otherwise only one devspace can run on a single docker host.